Update main figures 1, 2, and 4

jenna-tomkinson opened the pull request:
# Update main figures 1, 2, and 4

Per manuscript review, three main figures have been updated and are ready for final nit-picks!

jenna-tomkinson committed:
update main figures 1, 2, and 4

gwaybio reviewed (approved):
LGTM! Minor nitpicks (and I think one bug) below. No need for me to re-review

- Would it be easy to add the organelle to the genotype image montage? (in addition to stain name; I'm thinking something like `Nuclei\n(DAPI)`
- For the workflow figure, can you make a couple edits? 
  - Remove mention of plate information (ML box)
  - Add "holdout" to splitting box. Maybe it should also say "Balance and split" 🤷 
- Figure 2A still says "plate" as x axis label (`All plates` is fine as a tick label, the `Plate` as the x axis label is no longer accurate)
- I still see the `MeanIntensityEdge` montage as Figure 4C - this probably should be the actin montage? Oh wait, maybe the title is just wrong? It looks like the same cells from https://github.com/WayScience/NF1_SchwannCell_data_analysis/blob/b869c68be6b270ff1764eec642f0ad0b1a60d198/3.figures/main_figure_4/top_Null_feature_montage.png

jenna-tomkinson commented:
> Would it be easy to add the organelle to the genotype image montage? (in addition to stain name; I'm thinking something like Nuclei\n(DAPI)

Super easy, I have implemented this!

> For the workflow figure, can you make a couple edits?
> - Remove mention of plate information (ML box)
> - Add "holdout" to splitting box. Maybe it should also say "Balance and split" 🤷

Good points; I have updated this plot with both of these! 

> Figure 2A still says "plate" as x axis label (All plates is fine as a tick label, the Plate as the x axis label is no longer accurate)

This is fixed!

> I still see the MeanIntensityEdge montage as Figure 4C - this probably should be the actin montage? Oh wait, maybe the title is just wrong? It looks like the same cells from https://github.com/WayScience/NF1_SchwannCell_data_analysis/blob/b869c68be6b270ff1764eec642f0ad0b1a60d198/3.figures/main_figure_4/top_Null_feature_montage.png

Oh weird! That is a bug when generating the montages; I have fixed that. The title is fixed now for the actin feature.

jenna-tomkinson committed:
update + fix figures based on review

jenna-tomkinson commented:
@gwaybio Thanks for the review and feedback! I have updated these figures and will be merging now!
